doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -217,6 +219,13 @@ validate extended key usage information in certificates will need to define a
 custom "purpose" (see below) or supply a nondefault verification callback
 (L<X509_STORE_set_verify_cb_func(3)>).
 
+X509_STORE_CTX_set0_ocsp_resp() sets the OCSP response(s) for the verification
+of a certificate chain or for including in the TLS handshake, when the client
+requests OCSP stapling. The stack of OCSP responses I<sk> is not copied but
+just stored to the context.
+Ownership of the stack is transferred to I<ctx>, and so it should not be
+free'd by the caller.
```

Comment:
This is not the behavior of the current implementation. And as I wrote above the function should be renamed.